nixos/modules/installer/netboot/kexec-boot.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+
+kexecTree=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+initrd=$kexecTree/initrd
+kernelParams=$(<"$kexecTree/kernel-params")
+
+usage() {
+    printf >&2 'usage: %s [%s] [%s]\n' "$0" \
+        '--ssh-authorized-keys <file>' \
+        '--append <cmdline>'
+    exit 2
+}
+
+while (( $# )); do
+    arg=$1; shift 1
+    case "$arg" in
+        --ssh-authorized-keys)
+            (( $# )) || usage
+            authorizedKeysFile=$1; shift 1
```

Comment:
Do you think it would make sense to have some default authorized key files that are included by default? It's quite convenient to not have pass parameters. This could be also added in a later PR.

Reply:
Yes, possibly; I erred on the side of maintaining the current behaviour of a vanilla installer image by default. I also thought about transferring over the SSH host keys, but since a resulting NixOS installation would probably end up generating new keys anyway I thought it wasn't worth the bother.